runScript

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+sett -e
+#
+echo Hello Tomasz. Its our demo. Please enter a directory name:
+read foldername
+git clone https://github.com/smolamarcin/battleship $foldername
+cd "$foldername"
+mvn clean install;
+mvn test;
+
+#checkstyle
+mvn checkstyle:checkstyle;
+
```

Comment:
finbugs and jacoco reports are missing

Reply:
Findbugs and jacoco will be added in next release.